src/main/twirl/uk/gov/hmrc/play/views/helpers/singleCheckbox.scala.html

```diff
@@ -19,13 +19,13 @@
 @import views.html.helper._
 
 @elements = @{ new FieldElements(field.id, field, null, args.toMap, messages) }
-@value = @{ field.value match { case Some(x) => x case None => "false" case x => x }}
-<div class="form-field-single @if(elements.hasErrors) {error}">
+@value = @{ field.value match { case Some(x) => x case None => "false" case x => x } }
+@style = @{ elements.args.get('_inLine) match { case Some(true) => "form-checkbox" case _ => "block-label" } }
```

Comment:
I don't think we should have work that can be come different work by the flick of a switch (or argument). These are essentially different visual patterns that have similar underlying markup. Is the "simple checkbox" even a pattern that we should be using? You can just send in `form-checkbox`, `block-label` without having internal logic to provide these on a flag.